### PR TITLE
Update zos-lib for Solidity 0.5 support

### PR DESCRIPTION
**Pull request by hardlydifficult**
# Description

This latest release of zos-lib has updated the .sol files to allow for compiling with Solidity 0.5.  This update is a pre-req for resolving https://github.com/unlock-protocol/unlock/pull/892

# Checklist:

- [x] 1 PR, 1 purpose: my Pull Request applies to a single purpose
  - [x] This PR only contains configuration changes (package.json, etc.)
  - [ ] This PR only contains code changes (if configuration changes are required, do a separate PR first, then re-base)
- [x] My code follows the style guidelines of this project, including naming conventions
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] If my code adds or changes components, I have written corresponding stories with Storybook
- [x] I have performed a self-review of my own code
- [x] If my code involves visual changes, I am adding applicable screenshots to this thread

PS: [Read how to write the perfect pull request](https://blog.github.com/2015-01-21-how-to-write-the-perfect-pull-request/)

### CODE REVIEW

**Comment by nfurfaro** [comment]
@hardlydifficult  I'll review now. Any chance this version of `zos-lib` no longer depends on `openzeppelin-solidity`?

**Comment by nfurfaro** [comment]
To be honest, I'm not sure if we're even still using zos-lib in our smart contracts tests at this time.

**Comment by hardlydifficult** [comment]
Unlock.sol requires zos-lib for `Initializable`.

For the build fails, please help.  When I run the same from my account it passes: https://travis-ci.com/hardlydifficult/unlock/builds/96338689

**Comment by nfurfaro** [comment]
> Unlock.sol requires zos-lib for `Initializable`.
Ah yes, of course.

Not sure what's going on with the build though...

**Comment by nfurfaro** [comment]
@hardlydifficult RE: build fails
Are both of your PR's failing to build? I just submitted a tiny PR and it seemed to build fine.

**Comment by nfurfaro** [comment]
I'm pretty sure I don't have access to either the Netlify or Travis account settings, so I don't think I'll be able to help here, unfortunately. I'm pretty much only working on smart-contract related stuff.

**Comment by cellog** [comment]
hi @hardlydifficult thank you so much for these contributions! Our primary account holder with access to both Travis and netlify is indisposed, hang tight, we will get your PRs figured out and merged when he's back up and running. Sorry for that, and thanks for the patience!

**Comment by julien51** [comment]
Everyone, sorry about that. We added a way to push and pull docker images to make build faster... but it looks like the ENV variables required to do that are not used by travis on PR coming from forks (for security reasons).
I'm working on a way to skip this for external PRs. 
Please bear with me!

File: smart-contracts/package.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     "truffle-contract": "^3.0.6",
     "web3-utils": "^1.0.0-beta.36",
     "zos": "^v2.1.0-rc.0",
-    "zos-lib": "^2.0.2"
+    "zos-lib": "^v2.1.0-rc.0"
```
Comment (julien51) [inline smart-contracts/package.json]: cc @nfurfaro Is that the version *without* the heavy dependencies?

Comment (nfurfaro) [reply smart-contracts/package.json]: Yeah, zos-lib is tiny compared to zos. 29M vs 1071M

**Pull request merged**